pyuvdata/tests/test_fhd.py

```diff
@@ -54,22 +60,116 @@ def test_ReadFHDWriteReadUVFits():
     # check that a select on read works
     fhd_uv2 = UVData()
     uvtest.checkWarnings(fhd_uv2.read, [testfiles], {'freq_chans': np.arange(2)},
-                         message='Warning: select on read keyword set')
+                         message=['Warning: select on read keyword set',
+                                  'Telescope location derived from obs'],
+                         nwarnings=2)
 
     if not uvtest.scipy_warnings:
-        fhd_uv.read(testfiles)
+        uvtest.checkWarnings(fhd_uv.read, [testfiles], known_warning='fhd')
     else:
         # numpy 1.14 introduced a new deprecation warning.
         # Should be fixed when the next scipy version comes out.
         # The number of replications of the warning varies some and must be
         # empirically discovered. It it defaults to the most common number.
         n_scipy_warnings, scipy_warn_list, scipy_category_list = uvtest.get_scipy_warnings()
+        warn_list = ['Telescope location derived from obs'] + scipy_warn_list
+        category_list = [UserWarning] + scipy_category_list
         uvtest.checkWarnings(fhd_uv.read, [testfiles],
-                             message=scipy_warn_list, category=scipy_category_list,
-                             nwarnings=n_scipy_warnings)
+                             message=warn_list, category=category_list,
+                             nwarnings=n_scipy_warnings + 1)
```

Comment:
I'm actually wondering if we should ditch the scipy warning stuff -- it broke the Travis build quite awhile back, but that's in the past and they're only warnings...

Reply:
I'm a big fan of checking the warnings to be sure we're getting what we expect (not just because I wrote that function). The problem with dropping it is we would have to drop the other check warnings that you do simultaneously.
If it's too much work to refactor it, for this temporary warning, I'm ok if we leave it as-is and make an issue so we remember to get rid of it when scipy catches up.

Reply:
Scipy has caught up. So it's only executing one part of these if/else statements, I believe. I too think catching and checking warnings is critical, I'm just contemplating ripping out the scipy check, since it has caught up.

Reply:
Oh that's interesting. So the tests could fail for someone who has an unfortunately combination of numpy and scipy versions. But it should pass on travis if we just get rid of this? Hmm, I think it's probably fair to go ahead and drop the checks then, right?

Reply:
That's what I'm leaning toward...